I'm not really sure about this error from Shippable.

```2017-06-22 13:19:19 ERROR: Found 2 validate-modules issue(s) which need to be resolved:
2017-06-22 13:19:19 ERROR: lib/ansible/modules/cloud/openstack/os_quota.py:0:0: E309 version_added for new option (loadbalancer) should be 2.4. Currently 0.0 (75%)
2017-06-22 13:19:19 ERROR: lib/ansible/modules/cloud/openstack/os_quota.py:0:0: E309 version_added for new option (pool) should be 2.4. Currently 0.0 (75%)```

--- a/lib/ansible/modules/cloud/openstack/os_quota.py
+++ b/lib/ansible/modules/cloud/openstack/os_quota.py
@@ -99,6 +99,7 @@ options:
         required: False
         default: None
         description: Number of load balancers to allow.
+        version_added: "2.4"
     network:
         required: False
         default: None
@@ -111,6 +112,7 @@ options:
         required: False
         default: None
         description: Number of load balancer pools to allow.
+        version_added: "2.4"
     port:
         required: False
         default: None

